api/genomics/wdl/WgsCohortExtractPrep.wdl

```diff
@@ -78,50 +128,173 @@ task CreateCohortTable {
     preemptible: 3
     docker: "us.gcr.io/broad-gatk/gatk:4.1.8.0"
   }
-}
+} 
 
 
 task CreateCohortExtractTable {
 
   input {
     String cohort_uuid
+    String fq_cohort_sample_table
     String wgs_dataset
     String wgs_extraction_cohorts_dataset
     String wgs_extraction_destination_dataset
     String wgs_extraction_temp_tables_dataset
     String query_project
   }
-
+  
   command <<<
     set -e
-
+    
     echo "Exporting to ~{wgs_extraction_destination_dataset}.~{cohort_uuid}"
 
     python /app/ngs_cohort_extract.py \
       --fq_petvet_dataset ~{wgs_dataset} \
       --fq_temp_table_dataset ~{wgs_extraction_temp_tables_dataset} \
       --fq_destination_dataset ~{wgs_extraction_destination_dataset} \
       --destination_table ~{cohort_uuid} \
-      --fq_cohort_sample_names ~{wgs_extraction_cohorts_dataset}.~{cohort_uuid} \
+      --fq_cohort_sample_names ~{fq_cohort_sample_table} \
       --min_variant_samples 0 \
       --query_project ~{query_project} \
       --fq_sample_mapping_table ~{wgs_dataset}.metadata
-
-    echo ~{wgs_extraction_destination_dataset}.~{cohort_uuid} > cohort_extract_table.txt
+    
+    echo ~{wgs_extraction_destination_dataset}.~{cohort_uuid} > fq_cohort_extract_table.txt
   >>>
-
+  
   output {
-    String cohort_extract_table = read_string("cohort_extract_table.txt")
+    String fq_cohort_extract_table = read_string("fq_cohort_extract_table.txt")
   }
 
   runtime {
     memory: "3.75 GB"
     bootDiskSizeGb: "15"
     disks: "local-disk 10 HDD"
     preemptible: 3
-    # Temporary GATK that I'm using to get unblocked. We will realign with the mainstream GATK ah_var_store branch
```

Comment:
Comment still seems accurate - or can you use a standard image + gatk_override instead now?